gdeploycore/backend_setup.py

```diff
@@ -141,6 +141,8 @@ def parse_section(self, hostname):
         try:
             self.section_dict = Global.sections['backend-setup' +
                     hostname]
+            if '__name__' in self.section_dict.keys():
+                del self.section_dict['__name__']
```

Comment:
is __name__ added to dict.keys anywhere?

Reply:
This change is not needed. I happened to look into the issue 544 now. The conf file is written is wrong. If you have to use multiple vgs they have to be comma-separated. 
I'll comment on the issue.